data/rescue/rescue.file_list

```diff
@@ -48,6 +48,11 @@ TEMPLATE wicked|lvm2|syslog-service|util-linux|mdadm:
   /
   E postin
 
+# ensure /usr/share/pci.ids exists
+TEMPLATE hwdata:
+  /
+  e [ -e usr/share/pci.ids ] || ln -s pci.ids.d/pci.ids.dist usr/share/pci.ids
```

Comment:
could we make it fail if the target doesn't exist either? to make it a bit future proof